errors/interfaces/error.interface.ts

```diff
@@ -0,0 +1,6 @@
+export interface ErrorInterface {
```

Comment:
i think this should be like nestjs standard error response because i think frontend can handle them better this way